frontend/src/scenes/billing/billing-utils.ts

```diff
@@ -162,11 +162,13 @@ export const getUpgradeProductLink = (
     product: BillingProductV2Type,
     upgradeToPlanKey: string,
     redirectPath?: string,
-    includeAddons: boolean = true
+    includeAllAddons: boolean = true,
+    addonDict?: Record<string, string>,
+    billingLimit?: number
 ): string => {
-    let url = '/api/billing/activate?products='
+    let url = '/api/billing/activate?upgrade_products='
```

Comment:
Changed this to upgrade_products because it seemed to be conflicting with the products key in the redirectPath query param.